example/src/Screens/LinkingScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import { useUnhandledLinking } from '@react-navigation/native';
+import {
+  createStackNavigator,
+  StackScreenProps,
+} from '@react-navigation/stack';
+import React, { useContext } from 'react';
+import { Button, Platform, StyleSheet, Text, View } from 'react-native';
+
+const info = `
+\u2022 xcrun simctl openurl booted exp://127.0.0.1:19000/--/linking/profile
+\u2022 adb shell am start -W -a android.intent.action.VIEW -d "exp://127.0.0.1:19000/--/linking/profile" host.exp.exponent
+\u2022 http://localhost:19006/linking/profile
```

Comment:
fyi you can do `npx uri-scheme open exp://127.0.0.1:19000/--/linking/profile` with `--ios`/`--android`